### DISCUSSION TITLE
Found some .irx files from an old game I used to play if your intrested in them.

### DISCUSSION THREAD

**Discussion by 7dog123** [ps2dev/ps2sdk#700] [General]
Sorry if I didn’t post this in the right place for discussion, but I recently rediscovered an old game I used to play on the PS2 called Zapper: One Wicked Cricket. It includes some .irx files that aren’t found in the homebrew SDK. However, they are available in the official PS2 SDK. I can upload them if you’re interested.

**Comment by AKuHAK** [comment]
All game data files are protected by Sony copyright and cannot be distributed over the Internet.

**Reply by 7dog123** [reply]
Well, I could try to reverse engineer them at least part of the way and use the homebrew sdk as a reference to help get a base started. But have to figure out how to do just that.
